website/docs/d/scc_posture_scans_summary.html.markdown

```diff
@@ -0,0 +1,93 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_scc_posture_scan_summary"
+description: |-
+  Get information about scans_summary
+subcategory: "Security and Compliance Center"
+---
+
+# ibm_scc_posture_scan_summary
+
+Provides a read-only data source for scans_summary. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_scc_posture_scan_summary" "scans_summary" {
+	profile_id = "profile_id"
+	scan_id = "scan_id"
+}
+```
+
+## Argument Reference
+
+Review the argument reference that you can specify for your data source.
+
+* `profile_id` - (Required, String) The profile ID. This can be obtained from the Security and Compliance Center UI by clicking on the profile name. The URL contains the ID.
+* `scan_id` - (Required, Forces new resource, String) Your Scan ID.
+
+## Attribute Reference
+
+In addition to all argument references listed, you can access the following attribute references after your data source is created.
+
+* `id` - The unique identifier of the scans_summary.
+* `controls` - (Optional, List) The list of controls on the scan summary.
+Nested scheme for **controls**:
+	* `control_id` - (Optional, String) The scan summary control ID.
+	* `status` - (Optional, String) The control status.
+	  * Constraints: Allowable values are: pass, unable_to_perform
+	* `external_control_id` - (Optional, String) The external control ID.
+	* `control_desciption` - (Optional, String) The scan profile name.
+	* `goals` - (Optional, List) The list of goals on the control.
+	Nested scheme for **goals**:
+		* `goal_description` - (Optional, String) The description of the goal.
+		* `goal_id` - (Optional, String) The goal ID.
+		* `status` - (Optional, String) The goal status.
+		  * Constraints: Allowable values are: pass, fail
```

Comment:
If constraints part of status, we need to add Nested scheme..... and
Supported values are .... the values should be marked as bold